src/modules/dashboard/components/Extensions/WhitelistExtensionBody/WhitelistExtensionBody.tsx

```diff
@@ -1,7 +1,62 @@
 import React from 'react';
+import { FormattedMessage } from 'react-intl';
 
-const WhitelistExtensionBody = () => {
-  return <div>Content</div>;
+import {
+  ExtensionInitParams,
+  ExtensionParamType,
+} from '~data/staticData/extensionData';
+import { Textarea, ActionForm } from '~core/Fields';
+
+import { ActionTypes } from '~redux/index';
+
+import styles from './WhitelistExtensionBody.css';
+
+export interface FormValues {
+  agreement: string;
+}
+
+interface Props {
+  extensionInitParams: ExtensionInitParams[];
+  canInstall: boolean;
+}
+
+const WhitelistExtensionBody = ({ extensionInitParams, canInstall }: Props) => {
+  return (
+    <ActionForm
+      initialValues={{
+        agreement: undefined,
+      }}
+      submit={ActionTypes.COLONY_ACTION_GENERIC}
+      error={ActionTypes.COLONY_ACTION_GENERIC_ERROR}
+      success={ActionTypes.COLONY_ACTION_GENERIC_SUCCESS}
+    >
+      {() => (
+        <>
+          {extensionInitParams.map(
+            ({ paramName, title, description, type }) => (
+              <div key={paramName}>
+                {type === ExtensionParamType.Textarea && (
+                  <div className={styles.textArea}>
+                    <Textarea
+                      appearance={{ colorSchema: 'grey' }}
+                      label={title}
+                      name={paramName}
+                      disabled={!canInstall}
+                    />
+                    {description && (
+                      <p className={styles.textAreaDescription}>
+                        <FormattedMessage {...description} />
+                      </p>
+                    )}
+                  </div>
+                )}
+              </div>
+            ),
+          )}
+        </>
+      )}
+    </ActionForm>
```

Comment:
Nope all of this needs to be removed. We display init params in ExtensioSetup, there you need to put textarea (if ExtensionParamType === Textarea. This component will be used later on, when we will have extension enabled